aten/src/ATen/Context.h

```diff
@@ -102,6 +103,12 @@ class CAFFE2_API Context {
     });
   }
 
+  void lazyInitQInt() {
+    std::call_once(qint_init_, [&] {
```

Comment:
what's the plan with respect to hooks here?  Are we planning to build without certain quantized libraries?

Reply:
Removed this one as well